src/Illuminate/Validation/Validator.php

```diff
@@ -175,14 +175,14 @@ class Validator implements ValidatorContract
      *
      * @var array
      */
-    protected $sizeRules = ['Size', 'Between', 'Min', 'Max'];
+    protected $sizeRules = ['Size', 'Between', 'Min', 'Max', 'Gt', 'Lt', 'Gte', 'Lte'];
```

Comment:
is this right? Should it be `greater_than_or_equal' etc? Or is `Gte` etc correct here?

Reply:
yes you're right, should be greater_than_or_equal, etc